src/adk/api/qne_client.py

```diff
@@ -175,7 +175,7 @@ def _client_post(url: str, **params: ParametersType) -> Any:
 
```

Comment:
Error in pipeline:

With me, it only complains about the same 2 issues why pylint (pip install pylint==2.15.3) fails in the pipeline:
src\adk\api\qne_client.py:164:17: W3101: Missing timeout argument for method 'requests.get' can cause your program to hang indefinitely (missing-timeout)
src\adk\api\qne_client.py:171:17: W3101: Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely (missing-timeout)

I guess adding "timeout=10" as a parameter is oke. Waiting for 10 seconds is pretty long...

Reply:
Done.